### PR TITLE
Resize plane to fit data in `fit_plane_to_points`

### PR DESCRIPTION
**Pull request by user27182**
Implements feature #4650

### CODE REVIEW

File: pyvista/core/utilities/points.py [hunk 1]
```diff
@@ -182,14 +182,17 @@ def lines_from_points(points, close=False):
 def fit_plane_to_points(points, return_meta=False):
     """Fit a plane to a set of points using the SVD algorithm.
 
+    The plane is automatically sized and oriented to fit the extents of
+    the points.
+
```
Comment (tkoyama010) [inline pyvista/core/utilities/points.py:187]: @user27182
I believe that this change is a bug fix and that the previous behavior is inappropriate, is that correct? If you like, we will fix the pull request label to bug.

**Review by tkoyama010 — CHANGES_REQUESTED** [review]

File: pyvista/core/utilities/points.py (hunk 1 above)
Comment (user27182) [reply pyvista/core/utilities/points.py:187]: Yes I would agree it's more of a bug fix than a feature. Please go ahead and fix the label. Let me know if you need me to make any changes directly myself.

File: pyvista/core/utilities/points.py [hunk 2]
```diff
@@ -228,12 +234,65 @@ def fit_plane_to_points(points, return_meta=False):
...
     """
+    # Apply SVD to get orthogonal basis vectors to define the plane
     data = np.array(points)
-    center = data.mean(axis=0)
-    result = np.linalg.svd(data - center)
-    normal = np.cross(result[2][0], result[2][1])
-    plane = pyvista.Plane(center=center, direction=normal)
+    data_center = data.mean(axis=0)
+    U, S, Vh = np.linalg.svd(data - data_center)
```
Comment (tkoyama010) [inline pyvista/core/utilities/points.py:265]: Let's remove unused variables.
```suggestion
    _, _, Vh = np.linalg.svd(data - data_center)
```

**Review by tkoyama010 — DISMISSED** [review]
LGTM! Only one suggestion.

**Review by tkoyama010 — APPROVED** [review]

**Comment by tkoyama010** [comment]
If there are no other comments, this PullRequest will be merged after 24 hours.

**Pull request merged**